packages/cardpay-subgraph/schema.graphql

```diff
@@ -456,3 +458,48 @@ type TokenPair @entity {
   token1: Token!
   swaps: [TokenSwap]!             @derivedFrom(field: "tokenPair")
 }
+
+type RewardProgramRegistrationPayment @entity {
+  id: ID!
+  rewardProgramID: String!
+  admin: Account!
+  transaction: Transaction!
+  createdAt: BigInt!
+  prepaidCardPayment: PrepaidCardPayment!
+}
+
+type RewardeeRegistrationPayment @entity {
+  id: ID!
+  rewardProgramID: String!
+  rewardee: Account!
+  transaction: Transaction!
+  createdAt: BigInt!
+  prepaidCardPayment: PrepaidCardPayment!
+}
+
+type RewardSafe @entity {
+  id: ID!
+  rewardProgramID: String!
+  rewardee: Account!
+  safe: Safe!
+}
+
+type RewardeeClaim @entity {
+  id: ID!
+  rewardProgramID: String!
+  rewardee: Account!
+  rewardSafe: RewardSafe!
+  amount: BigInt!
+  transaction: Transaction!
+  timestamp: BigInt!
+}
+
+type RewardTokensAdd @entity {
```

Comment:
I feel like there should be a parent entity to this that I mentioned in a previous comment, `RewareProgram`, that has derived field that aggregraes all the `RewardTokensAdd` events.

Reply:
I also added claims field.